Objects/codeobject.c

```diff
@@ -152,11 +152,10 @@ PyCode_New(int argcount, int posonlyargcount, int kwonlyargcount,
     }
 
     n_varnames = PyTuple_GET_SIZE(varnames);
-    if (posonlyargcount + argcount <= n_varnames
+    if (argcount <= n_varnames
```

Comment:
Can it be merged into a single line?

Reply:
At the beginning of this function add the condition `argcount < posonlyargcount` instead of `argcount < 0`.